### PR TITLE
Add install documentation

### PR DESCRIPTION
**Pull request by cwansart**
Some documentation how to build and install the libary.

### CODE REVIEW

**Review by ArneLimburg — COMMENTED** [review]
Would you please move the "build" section to another file and link to that here.
I think it might be confusing for users to find that on the landing page (they may think, they have to do that, too).

**Review by ArneLimburg — CHANGES_REQUESTED** [review]
Would you please move the "build" section to another file and link to that here.
I think it might be confusing for users to find that on the landing page (they may think, they have to do that, too).

File: README.md [hunk 1]
```diff
@@ -4,4 +4,51 @@
...
+</dependency>
+```
+
+or when using Gradle the following line needs to be added to the dependencies block:
+
+```groovy
+implementation 'org.microjpa:microjpa:0.9.5'
+```
+
+The version listed here may not always be up-to-date. It can be checked on [Maven Central](https://search.maven.org/artifact/org.microjpa/microjpa).
```
Comment (ArneLimburg) [inline README.md:28]: Please remove this section. We should actually aim to always be up-to-date with the versions

**Comment by cwansart** [comment]
Good point, will do.

File: README.md (hunk 1 above)
Comment (cwansart) [reply README.md]: Done.

**Review by ArneLimburg — APPROVED** [review]

**Comment by ArneLimburg** [comment]
Improve documentation

**Pull request merged**